web_src/js/features/repo-issue.js

```diff
@@ -526,6 +527,18 @@ export function initRepoPullRequestReview() {
       editor.focus();
     }
   });
+
+  // helper function on our window object to update data
+  // in difftreestore
+  window.updateDiffTreeStore = function(pDiffFileInfo) {
+    DiffTreeStore.isIncomplete = pDiffFileInfo.isIncomplete;
+    DiffTreeStore.diffEnd = pDiffFileInfo.diffEnd;
+    DiffTreeStore.link = pDiffFileInfo.link;
+    DiffTreeStore.isLoadingNewData = pDiffFileInfo.isLoadingNewData;
+    DiffTreeStore.files.push(...pDiffFileInfo.files);
+  };
+  const pageData = window.config.pageData;
+  window.updateDiffTreeStore(pageData.diffFileInfo);
```

Comment:
I guess you could put the updater function also into `store.js` and export it there so you don't need to pollute `window`.

Reply:
i'm calling the update function from box.tmpl - that's why i need a global reference. Or is there any other approach i'm currently not aware of?

Reply:
Oh, I see. Yes for inline scripts, only this method can be used.